edx_notifications/server/web/static/edx_notifications/js/views/counter_icon_view.js

```diff
@@ -113,15 +114,19 @@ var CounterIconView = Backbone.View.extend({
      e.stopPropagation();
   },
 
-  autoRefreshNotifications: function(counterView) {
+ autoRefreshNotifications: function(counterView) {
      var currentModel = new CounterIconModel();
      currentModel.url = counterView.model.url;
      currentModel.fetch().done(function(){
          // if notification counter is incremented.
          if(currentModel.get('count') > counterView.model.get('count')){
-             var notification_alert = new Audio(counterView.view_audios.notification_alert);
              counterView.model = currentModel;
-             notification_alert.play();
+
```

Comment:
I'll merge this for now, so that Kashif can validate this fix, and I'll write up a new JIRA to make the audio optional.